packages/edit-site/src/components/block-editor/index.js

```diff
@@ -109,9 +99,13 @@ export default function BlockEditor( { setIsInserterOpen } ) {
 					headHTML={ window.__editorStyles.html }
 					head={ <EditorStyles styles={ settings.styles } /> }
 					ref={ ref }
-					contentRef={ contentRef }
+					contentRef={ mergedRefs }
 				>
-					<Canvas body={ contentRef } />
```

Comment:
Notice how we no longer need a separate component, because the hook can be called before the content is mounted.